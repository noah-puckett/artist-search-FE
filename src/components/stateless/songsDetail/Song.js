import React from 'react';
import PropTypes from 'prop-types';

function Song({ title }) {
    return (<>
        <h4>{title}</h4>
    </>
    );
}

Song.propTypes = {
    title: PropTypes.string.isRequired,
};

export default Song;
