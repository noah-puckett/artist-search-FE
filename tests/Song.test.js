import React from 'react';
import { shallow } from 'enzyme';
import Song from '../src/components/stateless/songsDetail/Song';

describe('Lyrics component', () => {
    it('renders Song', () => {
        const wrapper = shallow(<Song title='hello' />);
        expect(wrapper).toMatchSnapshot();
    });
});
