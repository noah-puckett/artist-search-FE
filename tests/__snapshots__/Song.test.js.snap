// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Song 1`] = `
<Fragment>
  <h4>
    hello
  </h4>
</Fragment>
`;
